lib/java/com/google/android/material/color/MaterialColorUtils.java

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.material.color;
+
+import android.content.Context;
+import android.view.View;
+import androidx.annotation.ColorInt;
+import androidx.annotation.NonNull;
+import com.google.android.material.R;
+
+/** Utility methods for {@link MaterialColors}. */
+public class MaterialColorUtils {
```

Comment:
I think it will be better to define this as an enum class under MaterialColors, say, MaterialColors.Color.PRIMARY, etc., and provide getColor() method for the enum class.